webclient/src/views/view/view-view.scss

```
@@ -341,12 +341,27 @@
     #roomfinder-map-img {
         width: 100%;
         display: block;
+        cursor: pointer;
     }
 
     #roomfinder-map-select > label {
         padding: .05rem .3rem;
     }
 
+    #roomfinderModal-container {
+        padding-bottom: 4em;
+    }
+
+    #modalRoomfinder-map-cross {
+        position: absolute;
+        transition: transform 1s;
```

```suggestion
        transition: transform .3s;
```
I dont think that this being slower is better. We should merge this with `#roomfinder-map-cross`